src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -21,6 +23,13 @@ import { SearchControls } from './search';
  */
 const SUB_CATEGORY_SHORT_SIZE = 10;
 
+// Organisms for which we display tabs
+// Names for the tabs are hard-coded because we want to display disabled tabs for tabs for which there are no results
+// Mapping of shortened organism name and full scientific organism name is used for human and mouse thumbnail images
+const organismTerms = {
```

Comment:
The syntax is correct. However, it seems like the key and values are reverse. I normally see the key without a space and something sort and recognizable. The value is what gets more complicated.

Reply:
I think I can completely eliminate the need for this to be an object, by just renaming the image files. I think that's the simplest solution here.

Reply:
I think there's no need for this to be an object at all - I can just use an array and that will be simpler. Just need to rename the pngs.